WORKSPACE

```diff
@@ -53,6 +53,16 @@ npm_import(
 translate_package_lock(
     name = "npm_deps",
     package_lock = "//example:package-lock.json",
+    patch_args = {
+        "@gregmagolan/test-a@0.0.1": [
+            "-p1",
+        ],
+    },
+    patches = {
+        "@gregmagolan/test-a@0.0.1": [
+            "//example:test-a.patch",
+        ],
+    },
```

Comment:
Planning to also support wildcards `"*"` to patch everything (ngcc case), and the package name minus the version to patch all version. But will do that in a separate PR.

Reply:
Actually I think the wildcard is only applicable for postinstall, not patches.